src/constants/theme.js

```diff
@@ -73,6 +73,7 @@ const fontSizes = {
   H6: 9,
   LeadParagraph: 16,
   Paragraph: 14,
+  Faq: 13,
```

Comment:
Theme is meant for general props. Typography in particular is supposed to match the corresponding part in the design system: https://www.figma.com/file/N4Xbl652BhzOutXmzhcrLGch/DS-05-%2F-UI-%2F-Atoms?node-id=9%3A10

All FAQs are normally implemented in the FAQ component so there's no need for a global settings on this one, we can set the font size directly on the FAQ component.

Reply:
Ah ok cool good to know. The FAQ was using those global settings before but I wanted to increase a bit the font size because it was too small.